### PR TITLE
add my card information - emily lubkert's card

### PR DESCRIPTION
**Pull request by emilylubkert**
added a new card and modified the content with my info

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -108,6 +108,34 @@ <h2>A project for first-time contributions</h2>
...
+                </a>
+              </li>
+              <li>
+                <a href="https://github.com/lydiahallie/javascript-questions#readme" target="_blank" title="Third resource">
+                  JavaScript Questions
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
```
Comment (dilshankarunarathne) [inline index.html:139]: Please keep the comment 
`<!-- ________*END TEMPLATE* Contributor card END ________ -->`  and change it to
to `<!-- ________ Your_Name Contributor card END ________ -->`

File: index.html [hunk 2]
```diff
@@ -108,6 +108,34 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
+        <div class="card">
```
Comment (dilshankarunarathne) [inline index.html:111]: Please keep the comment 
`<!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________ -->` and change it to
into `<!-- ________ Your_Name Contributor card START ________ -->`

File: index.html [hunk 3]
```diff
@@ -108,6 +108,34 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
```
Comment (dilshankarunarathne) [inline index.html:110]: Please leave an empty line between the template and your card. That would make it easier for everybody to identify and understand the code.

**Review by dilshankarunarathne — CHANGES_REQUESTED** [review]
Hello @emilylubkert, thank you for contributing to this project. ❤
But there are a couple of small changes that need to be done before we go ahead and merge your code.
Once you've made those changes, please get back to us. We will be able to review and merge your pull request in a flash.

File: index.html (hunk 2 above)
Comment (emilylubkert) [reply index.html:113]: Got it!

**Review by dilshankarunarathne — APPROVED** [review]

**Comment by dilshankarunarathne** [comment]
Congratulations 🎉❤ @emilylubkert.
Your pull request was merged. 
You will be able to see your card on the [project website](https://syknapse.github.io/Contribute-To-This-Project/).

**Comment by emilylubkert** [comment]
🎉 Thank you!

**Pull request merged**